packages/polaris-viz-core/src/hooks/useFilteredSparkLineData.ts

```diff
@@ -0,0 +1,30 @@
+import {useMemo} from 'react';
+import type {DataSeries} from 'types';
+
+export function useFilteredSparkLineData(data: DataSeries[]) {
+  const allNumbers = useMemo(() => {
+    if (data.length <= 1) {
+      return [];
+    }
+
+    return data.reduce<number[]>((prev, cur) => {
+      const numbers = cur.data
+        .map(({value}) => value)
+        .filter((value) => value !== null) as number[];
+      return prev.concat(...numbers);
+    }, []);
+  }, [data]);
+
+  const areAllZeros = useMemo(() => {
+    return !allNumbers.some((num) => num !== null && num !== 0);
```

Comment:
super nit: I feel like using `every` here would be easier to follow
```suggestion
    return allNumbers.every((num) => num === null || num === 0)
```

Reply:
Ohhh interesting 👂 👍